tests/integration/components/bs-form/element/errors-test.js

```diff
@@ -26,4 +26,11 @@ module('Integration | Component | bs form/element/errors', function(hooks) {
     assert.dom(`.${formFeedbackClass()}`).exists({ count: 1 });
     assert.dom(`.${formFeedbackClass()}`).hasText('foo');
   });
+
+  test('shows multiple errors', async function(assert) {
+    this.set('messages', A(['foo', 'bar']));
+    await render(hbs`{{bs-form/element/errors messages=messages show=true showMultipleErrors=true}}`);
+    assert.dom(`.${formFeedbackClass()}`).exists({ count: 1 });
```

Comment:
This is failing, should be `{ count: 2 }` now! See https://travis-ci.org/kaliber5/ember-bootstrap/jobs/325103870#L1027